PipeableTests/ScriptTests.swift

```diff
@@ -0,0 +1,24 @@
+@testable import Pipeable
+import XCTest
+
+final class ScriptTests: XCTestCase {
+    func test_goto() throws {
+        let result = runScript("""
```

Comment:
How does this work in Swift if it's not async? Does it just block the calling thread? 

Not sure what we want here in terms of API -- sync or async? I was defaulting to async, but wondering if you have other thoughts.

Reply:
Currently, it blocks the main thread as it is sync API. I should be `async`, I will get that fixed before merging the PR.